lib/langs/pt-BR.js

```diff
@@ -4,10 +4,15 @@ module.exports = {
   posts: 'Posts',
   category: 'Categoria',
   categories: 'Categorias',
-  allCategories: 'Todas',
   tag: 'Tag',
   tags: 'Tags',
 
+  // post filter
+  allCategories: 'Todas',
+  search: 'Pesquisar',
+  searchHint: 'Pesquise no título e conteúdo',
+  noRelatedPosts: 'No related posts',
```

Comment:
Ping @tgmti  😄

Reply:
Allways

    noRelatedPosts: 'Nenhum artigo encontrado',

while you're at it, could you, please change these others?

    home: 'Início',
    posts: 'Artigos',
    prevPost: 'Anterior',
    nextPost: 'Próximo',
    notFound: 'Opa! A página que você está procurando não está aqui.',

Sorry, 
i'm changing my notebook, and no able to send pr this week.

Reply:
Never mind~ Changed~